README.md

```diff
@@ -1,76 +1,231 @@
+        </li>
+        <li>
+            <a href="https://rxjs.dev/"><img src="https://img.shields.io/badge/-RxJs-purple?style=for-the-badge&logo=ReactiveX" alt="RxJs"></a>
+        </li>
+        <li>
+            <a href="https://tailwindcss.com/"><img src="https://img.shields.io/badge/-Tailwind%20CSS-white?style=for-the-badge&logo=Tailwind%20CSS" alt="TailWind CSS"></a>
+        </li>
+    </ul>
+</div>
+<br>
+
+<!--CONTRIBUTING-->
+<div>
+    <h2><b>Contribution</b></h2>
+</div>
+
+<div align="center">
+    <h3><b><u>Main Developer</u></b></h3>
+    <br>
+    <a href="https://github.com/Zagrios"><img src="https://images.weserv.nl/?url=avatars.githubusercontent.com/u/40181755?v=4&h=150&w=150&fit=cover&mask=circle&maxage=7d"></a>
+    <br><br>
+    <h3><b><u>Support Developer</u></b></h3>
+    <br>
+    <a href="https://github.com/Iluhadesu"><img src="https://images.weserv.nl/?url=avatars.githubusercontent.com/u/92525749?v=4&h=150&w=150&fit=cover&mask=circle&maxage=7d"></a>
+    <a href="https://github.com/GaetanGrd"><img src="https://images.weserv.nl/?url=avatars.githubusercontent.com/u/40648115?v=4&h=150&w=150&fit=cover&mask=circle&maxage=7d"></a>
+</div>
+<div align="right">
+    [<a href="#readme-top">Return to top</a>]
+</div>
+<br>
+
+<!--USAGE-->
+<div align="center">
+    <h1> </h1>
+    <h1><b>Usage</b></h1>
+</div>
+
+<!--HOW TO INSTALL?-->
+<div>
+    <h2><b>How to install?</b></h2>
+    <ul>
+        <li>Download the <a href="https://github.com/Zagrios/bs-manager/releases/latest">latest release</a> from <a href="https://github.com/Zagrios/bs-manager/releases">Releases</a>.</li>
+        <li>Execute the installer and <a href="../..">BSManager</a> will start itself.</li>
+        <li>Once its done if you want to download a version, select a version and download it! </li>
+        <li>From the version page you can choose options, launch the version, and more!</li>
+</div>
+<br>
+
+<!--MANAGE VERSIONS-->
+<div>
+    <h2><b>Manage Versions</b></h2>
+    <span>
+        <a href="https://github.com/Zagrios/bs-manager">BSManager</a> allows you to manage your versions of Beat Saber. It gives you the ability to download any version of Beat Saber as long as you have a Steam account that already owns the game. Using <a href="https://github.com/Zagrios/bs-manager">BSManager</a>, you can easily manage and update your collection of Beat Saber versions.
+    </span>
+</div>
+<br>
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213008824-25cfdda3-6350-42dc-aa53-2132b4b8e838.mov
+</div>
+
+<!--CUSTOM UR INSTANCES-->
+<div>
+    <h2><b>Customize your instances</b></h2>
+    <span>
+        The tool will allow you to completely customize each of your downloadable versions, you can change the color of the icon, choose a name for the instance and finally install your mods and map according to the Beat Saber version! 
```

Comment:
🔴 each of your "downloaded" versions
🟡 To tool will ... -> BSManager ... (no will here)
🟡 Choose a name for the "version" (instance is a technical term)
🟡 mods, maps, and other contents  ....

Reply:
fixed